src/main/database.cpp

```diff
@@ -21,6 +22,20 @@ using namespace kuzu::transaction;
 namespace kuzu {
 namespace main {
 
+void getFileFlags(SystemConfig systemConfig, int& flags, FileLockType& lock, bool create_new) {
```

Comment:
`create_new` -> `createNew`. Be careful with your code style. We follow camelCase, and functions start with lower case char.